modules/sdk-core/src/bitgo/staking/iStakingWallet.ts

```diff
@@ -21,12 +21,16 @@ export interface StakingRequest {
 export interface StakeOptions {
   amount: string;
   clientId?: string;
+  validator?: string;
+  durationSeconds?: string;
+  [otherOptions: string]: unknown;
```

Comment:
this kind of breaks the strict typing we prefer to use. Can you elaborate on the use case here?